examples/py-hypertrace/workflow.py

```diff
@@ -1,66 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Moved to `isofit/test/test_hypertrace.py`, and invoked in the `pip` package tests.